guides/V1_GetActionContext.md

```diff
@@ -0,0 +1,86 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'get-action-context'
+is: 'experimental'
+---
+
+
+# Get action context information 
+
+If you have registered a Client action handler callback for your App, then when your App is called through this callback, 
+it will receive an action handler context token. Your App needs to use this token with this API to extract the proper 
+action context.
+
+### Path
+```
+GET /v1/apps/{appId}/getActionContext/{actionHandlerContextToken}
+```
+**actionHandlerContextToken** has only 5 minutes lifetime, and can only be used once. It will be deleted once it is used. 
+
+#### Parameters
+
+|Type|Name|Description|Schema|
+|---|---|---|---|
+|**Header**|**Authorization** <br>_required_ |Authorization header, in form of Bearer {access_token}. The access_token is issued as result of an oauth flow by Watson Work Services|String|
+|**Path**|**appId** <br>_required_ |Unique id of an app|String|
+|**Path**|**actionHandlerContextToken** <br>_required_|token generated by caller before|String|
+
+#### Responses
+
+|HTTP Code|Description|Schema|
+|---|---|---|
+|**200**|The configuration data is returned and deleted successfully.|[ConfigurationData](#action-output-entity)|
```

Comment:
Reference to `configuration data` is left over from config flow.  What is returned is the actionContext data which varies by type of action.